# MDK parameter of the admittance controller

# MDK parameter
float64[36] M
float64[36] D
float64[36] K

